integration/dkg/dkg_whiteboard_client.go

```diff
@@ -46,8 +50,15 @@ func (wc *WhiteboardClient) ReadBroadcast(fromIndex uint, referenceBlock flow.Id
 
 // SubmitResult implements the DKGContractClient interface. It publishes the
 // DKG results under the node's ID.
-func (wc *WhiteboardClient) SubmitResult(groupKey crypto.PublicKey, pubKeys []crypto.PublicKey) error {
-	wc.whiteboard.submit(wc.nodeID, groupKey, pubKeys)
+func (wc *WhiteboardClient) SubmitResult(groupKey crypto.PublicKey, pubKeys []crypto.PublicKey, indexMap flow.DKGIndexMap) error {
+	wc.whiteboard.submit(wc.nodeID, groupKey, pubKeys, indexMap)
+	return nil
+}
+
+// SubmitEmptyResult implements the DKGContractClient interface. It publishes the
+// empty DKG result under the node's ID.
+func (wc *WhiteboardClient) SubmitEmptyResult() error {
```

Comment:
If you agree with the suggested documentation and naming changes of `SubmitResult` and `SubmitEmptyResult`, please also update here.